cmd/kops/create_cluster.go

```diff
@@ -1014,6 +1014,9 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 	if cluster.Spec.API.LoadBalancer != nil && cluster.Spec.API.LoadBalancer.Type == "" {
 		switch c.APILoadBalancerType {
 		case "", "public":
+			if cluster.Spec.Topology.Masters == api.TopologyPrivate {
```

Comment:
@mikesplain Will there be folks out there that really wants to put a public ELB in front of private masters?

It should be --api-loadbalancer-type='internal' in the error message.

Reply:
@Sprinkle Ahh yes good eye, thanks! Fixed the typo.  You can't put a public ELB in front of a private topology without public subnets... It currently just doesn't work and spins up a cluster anyway, so I thought an error message would be helpful. If I'm missing a case though, let me know :)